src/extension/wps/src/test/resources/config.properties

```diff
@@ -0,0 +1,21 @@
+urlMangler.downloadUrl.key = ^
+urlMangler.downloadUrl.value = http://data.aodn.org.au/
+
+csiro_cars_weekly_url.filename = CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_Australia_weekly.nc
+csiro_cars_monthly_url.filename = CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_World_monthly.nc
+
+# SRS Layer
+srs = srs
+# Semicolon separated values
+srs.ncks.parameters = --mk_rec_dmn;time;-v;time,lat,lon,dt_analysis,l2p_flags,quality_level,satellite_zenith_angle,sea_surface_temperature,sses_bias,sses_count,sses_standard_deviation,sst_dtime
```

Comment:
Can identify whether --mk_rec_dmn is necessary by testing ucar.nc2.Variable.isUnlimited on the time variable.  False means --mk_rec_dmn is necessary